src/main/resources/assets/trafficcontrol/misc/signs.json

```diff
@@ -3936,6 +3932,452 @@
     "type": "triangle",
     "variant": "95"
   },
+ {
```

Comment:
This whole inserted area overlap with already existing variant numbers and therefore isn't doing anything.  I'm not 100% sure what you're trying to do here, either.  It seems like this is a mix of signs that have already been added but have a different variant number or you're trying to add new signs, which 1) are missing textures in this PR, and 2) should be done through other avenues rather than done through a PR, though I appreciate you trying to help put them in.